stdlib/Profile/src/Profile.jl

```diff
@@ -1250,8 +1250,10 @@ end
 
 
 """
-    Profile.take_heap_snapshot(filepath::String, all_one::Bool=false, streaming=false)
-    Profile.take_heap_snapshot(all_one::Bool=false; dir::String, streaming=false)
+    Profile.take_heap_snapshot(filepath::String, all_one::Bool=false,
+                               redact_data::Bool=false; streaming::Bool=false)
+    Profile.take_heap_snapshot(all_one::Bool=false, redact_data:Bool=false;
+                               dir::String=nothing)
```

Comment:
Ah, thanks for fixing the docstring to reflect the kwarg correctly.

I think `redact_data` should be a kwarg not a positional argument as well?

I think all_one should really have been a kwarg too, but that was added in a different PR and I missed the review on that one, and now it should probably stay how it is for compatability. But otherwise, i think all new params here should mostly likely be keyword arguments.

Reply:
Made `redact_data` a kwarg.